package.json

```diff
@@ -25,7 +21,8 @@
   },
   "devDependencies": {
     "eslint-config-schibsted": "file:./packages/eslint-config-schibsted",
-    "eslint-config-schibsted-node": "file:./packages/eslint-config-schibsted-node"
+    "eslint-config-schibsted-node": "file:./packages/eslint-config-schibsted-node",
```

Comment:
you shouldn't need these as yarn hoists them up automatically

Reply:
Ah yes, with the workspaces. Patch incoming.